web_src/css/repository.css

```diff
@@ -2120,17 +2124,13 @@
   margin-bottom: 15px;
 }
 
-.repository.wiki.view > .markup {
-  padding: 15px 30px;
+.file-view.markup {
+  padding: 1em 2em;
 }
 
-.repository.wiki.view > .markup h1:first-of-type,
-.repository.wiki.view > .markup h2:first-of-type,
-.repository.wiki.view > .markup h3:first-of-type,
-.repository.wiki.view > .markup h4:first-of-type,
-.repository.wiki.view > .markup h5:first-of-type,
-.repository.wiki.view > .markup h6:first-of-type {
-  margin-top: 0;
```

Comment:
These are dead code, `.markup` is no direct child of `.view` probably since a very long time.